crime-commons-spring-boot-starter-rest-client/src/main/java/uk/gov/justice/laa/crime/commons/common/ErrorDTO.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.laa.crime.commons.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.Value;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+@AllArgsConstructor
+@NoArgsConstructor(force = true)
+@Builder
+public class ErrorDTO extends Throwable {
+    String traceId;
+    String code;
+    String message;
+}
```

Comment:
Whats the reason for not using the ErrorDTO in commons classes? Is it to try and avoid dependencies between the sub modules of commons library?

Reply:
Correct

Reply:
Ok thats fine, just wanted to double check 😄